src/Voyager.php

```diff
@@ -77,6 +77,21 @@ public static function can($permission)
         }
     }
 
+    public static function for($permission)
```

Comment:
NOTE TO SELF: Remove this method and use `Voyager::can` instead when merging to `v0.11`.

Since in `v0.11` the method `Voyager::can` no longer throws exception.